src/get-started/wordcount-example.md

```diff
@@ -578,15 +579,15 @@ public class DebuggingWordCount {
 
 #### Direct Runner
 
-If you execute your pipeline using `DirectRunner`, it will print the log messages directly to your local console.
+If you execute your pipeline using `DirectRunner`, it prints the log messages directly to your local console.
```

Comment:
This is usually the case, but requires `Slf4j` to be set up appropriately

Reply:
added sentence for Java about adding Slf4j to class path